postmark.php

```diff
@@ -51,47 +53,82 @@ function load_settings() {
         return json_decode( $settings, true );
     }
 
-
     function admin_menu() {
         add_options_page( 'Postmark', 'Postmark', 'manage_options', 'pm_admin', array( $this, 'settings_html' ) );
     }
 
+    // Retrieves additional logs.
+    function postmark_load_more_logs() {
+
+      global $wpdb;
+
+      $new_rows_html = "";
+
+      $table = $wpdb->prefix . "postmark_log";
+
+      // Checks the wp_nonce.
+      if ( ! isset($_POST['_wpnonce']) || ! wp_verify_nonce( $_POST['_wpnonce'], 'postmark_nonce' ) ) {
+        wp_die(__('We were unable to verify this request, please reload the page and try again.'));
+      }
+
+      // Retrieves more logs from logs table using offset and prepare() to prevent SQL injections.
+      $result = $wpdb->get_results( $wpdb->prepare("SELECT * FROM $table ORDER BY time DESC LIMIT %d OFFSET %d", 10, $_POST['offset']));
```

Comment:
I'm not familiar, does the `prepare` method parameterize this statement to prevent SQL injection?